### PR TITLE
Implementation of new cuts added to HelixFinders and new cpr and apr trig sequences

### PR DESCRIPTION
**Pull request by ethanmar**
- Added maxEDepAvg and maxNHitsRatio to corresponding filter/producers, in correspondence with Offline PR: [https://github.com/Mu2e/Offline/pull/1289](url)
- Added cprDeHighP and aprHighP trigger sequences which have looser constraints on impact parameter, lambda, and tanDip

### CODE REVIEW

File: core/trigFilters.fcl [hunk 1]
```diff
@@ -47,6 +50,7 @@ TrigCprFilters:{
       maxPanelToHelixDPhi: 1.309
       maxXDPhi: 5
       maxZTripletSearch: 0
+      bz: 0.0
```
Comment (gianipez) [inline core/trigFilters.fcl:53]: this should be removed

File: core/trigFilters.fcl [hunk 2]
```diff
@@ -58,7 +62,7 @@ TrigCprFilters:{
       nHitsMaxPerPanel: 1
       plotall: false
       sigmaPhi: 1.636e-1
-      smartTag: 1
+      # smartTag: 1
```
Comment (gianipez) [inline core/trigFilters.fcl:65]: you can delete this line

File: core/trigFilters.fcl [hunk 3]
```diff
@@ -68,7 +72,7 @@ TrigCprFilters:{
       zWeights: false
     }
     StrawHitCollectionLabel: "TTflagBkgHits"
-    StrawHitFlagCollectionLabel: "DeltaFinder:ComboHits"
+    # StrawHitFlagCollectionLabel: "DeltaFinder:ComboHits"
```
Comment (gianipez) [inline core/trigFilters.fcl:75]: delete this line please

**Comment by ethanmar** [comment]
Addressed Giani's comments

**Comment by ethanmar** [comment]
- Centralized the EDepAvg cut, which is now also added to MPR
- Added a tprHighP trigger sequence with looser cuts on lambda, d0, and tanDip

**Comment by ethanmar** [comment]
- Updated configuration of maxedepavg cut
- Complimentary changes to Offline can be found at following PR: [https://github.com/Mu2e/Offline/pull/1289](url)

**Comment by brownd1978** [comment]
I see several new highp (no stoptarg) lines were added. Are these intended to replace the highp_stoptarg lines?

**Comment by ethanmar** [comment]
@gianipez will give a better answer than me. I'm not sure if they're intended to replace highp_stoptarg lines, but for now we're evaluating the rates of these highp sequences (with looser cuts related to the stoptarg) to see if they are comparable to the highp_stoptarg sequences.

**Comment by ethanmar** [comment]
Commented out the new highP trigger lines which will be uncommented in a separate PR

**Review by brownd1978 — APPROVED** [review]

**Comment by brownd1978** [comment]
Are any changes still coming? If not I will merge this and Offline 1289

**Comment by ethanmar** [comment]
No, everything is ready to go.

**Pull request merged**